.gitignore

```diff
@@ -250,4 +250,21 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
+
+
+# IDEs
+.idea
```

Comment:
.idea is already above

Reply:
folder .idea isn't above. Above are only some files inside it. Before I added this `git status` was showing me this folder.